frontend/src/Pages/ProgramOverviewDashboard.tsx

```diff
@@ -0,0 +1,303 @@
+            },
+            facilitator: 'Mark Smith',
+            start_date: '2021-09-01',
+            end_date: '2021-09-30',
+            enrolled_residents: 16,
+            capacity: 40
+        }
+    ]
+};
+
+export default function ProgramOverview() {
+    const { id } = useParams<{ id: string }>();
+    const [page, setPage] = useState(1);
+    const [perPage, setPerPage] = useState(20);
+    const [selectedSections, setSelectedSections] = useState<number[]>([]);
+    const [searchTerm, setSearchTerm] = useState<string>('');
+    const [sortQuery, setSortQuery] = useState<string>('name_last asc');
+    const [includeActive, setIncludeActive] = useState(true);
+    const { data: programResp, error: programError } = useSWR<
+        ServerResponseOne<Program>,
+        AxiosError
+    >(`/api/programs/${id}`);
+    const program = programResp?.data;
+
+    const meta = {
+        current_page: page,
+        per_page: perPage,
+        total: sections.data.length,
+        last_page: Math.ceil(sections.data.length / perPage)
+    };
+
+    if (programError) {
+        <Error />;
+    }
+
+    const allSelected =
+        selectedSections.length === sections.data.length &&
+        sections.data.length > 0;
+
+    function handleToggleAll(checked: boolean) {
+        if (checked) {
+            setSelectedSections(sections.data.map((s) => s.id));
+        } else {
+            setSelectedSections([]);
+        }
+    }
+
+    function handleToggleRow(sectionId: number) {
+        setSelectedSections((prev) =>
+            prev.includes(sectionId)
+                ? prev.filter((id) => id !== sectionId)
+                : [...prev, sectionId]
+        );
+    }
+
+    const handleChange = (newSearch: string) => {
+        setSearchTerm(newSearch);
+        setPage(1);
+    };
+
+    const handleSetPerPage = (val: number) => {
+        setPerPage(val);
+        setPage(1);
+
+        setSortQuery(sortQuery); //Just to get eslint to stop complaining remove when this is is built out
+    };
+
+    return (
+        <div className="p-4">
+            <h1 className="text-2xl font-bold mb-2">{program?.name}</h1>
+            <p className="mb-4">{program?.description}</p>
+
+            <div className="flex gap-4 mb-8 items-start">
+                <div className="flex-none w-[220px]">
+                    <StatsCard
+                        title="Residents Enrolled"
+                        number="104"
+                        label="residents"
+                        tooltip="Placeholder data"
+                    />
+                </div>
+
+                <div className="flex-none w-[220px]">
+                    <StatsCard
+                        title="Overall Completion"
+                        number="50"
+                        label="%"
+                        tooltip="Placeholder data"
+                    />
+                </div>
+
+                <div className="flex-1 card p-2 flex flex-col">
```

Comment:
Can we make this a stacked bar chart for the past 6 months? Values would be Dropout vs. Completions over a rolling period.

Completions are the top segment of the bar.
Dropouts are the bottom segment of the bar.